emission/net/ext_service/gmaps/common.py

```diff
@@ -23,46 +23,72 @@ def get_recommender_uuid_list():
     return edb.get_trip_db().find({}).distinct("user_id")
 
 def coerce_gmaps_time(time):
-	lst = time.split()
-	if len(lst) == 4:
-		return timedelta(hours=int(lst[0]), minutes=int(lst[2]))
-	elif len(lst) == 2:
-		return timedelta(minutes=int(lst[0]))
+    lst = time.split()
+    if len(lst) == 4:
+        return timedelta(hours=int(lst[0]), minutes=int(lst[2]))
+    elif len(lst) == 2:
+        return timedelta(minutes=int(lst[0]))
 
-def google_maps_to_our_trip(google_maps_json, _id, user_id, trip_id, mode, org_start_time):
+def google_maps_to_our_trip_list(google_maps_json, _id, user_id, trip_id, mode, org_start_time):
```

Comment:
why do we have two functions - one for a trip list and one for a trip, that are completely different? I would expect one of them to call the other...